frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -792,25 +803,26 @@
                     utilities.sendRequest(parameters);
                 }, 5000);
             };
-
+            
             vm.stopFetchingSubmissions = function() {
                 $interval.cancel(vm.poller);
             };
             vm.stopFetchingSubmissions();
             vm.isResult = true;
             if (phaseId !== undefined) {
                 vm.phaseId = phaseId;
+                vm.mySubmissionPhaseSlug = phaseId;
```

Comment:
@Kajol-Kumari I think you have to work a liitle bit more here as now from the function `getResults`, its argument coming as slug not phase id in variable `phaseId` so you need to stop setting to controller variable `vm.phaseId` in line [813](https://github.com/Cloud-CV/EvalAI/pull/2786/files?file-filters%5B%5D=.js&file-filters%5B%5D=.py#diff-14a845646ed48757b5c7798b7ccdddb6R813) and also make sure that it is not breaking other functionalities where we might not able to get `vm.phaseId` after the removal of line 813.

Reply:
@Sanji515 we need to keep it as we will require it for download. Also as the submission extraction part is being done using slug. So, i think i need to change it for the `download_submission` also as now the `download_all_submission` will receive `slug` instead of the `id`